dapps/assets/src/components/contacts/InviteDispatcher.ts

```diff
@@ -0,0 +1,160 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+import * as bcc from '@evan.network/api-blockchain-core';
+import * as dappBrowser from '@evan.network/ui-dapp-browser';
+import { Dispatcher, DispatcherInstance } from '@evan.network/ui';
+
+const dispatcher = new Dispatcher(
+  `assets.${dappBrowser.getDomainName()}`,
+  'inviteDispatcher',
+  40 * 1000,
+  '_assets.contacts.invite-dispatcher'
+);
+
+/**
+ * Return a new bcc.Profile instance for a given accountId.
+ *
+ * @return     {bcc.Runtime}  profile for account
+ */
+const getProfileForAccount = (runtime: bcc.Runtime, accountId: string) => {
+  return new bcc.Profile({
```

Comment:
Could be also done by destructuring. But it think it was just taken from the old code. ;)

```
return new bcc.Profile({
  ...runtime,
  accountId
})
```

Reply:
Your suggested code throws errors. I'm not sure how to do it like you intend it to.

Reply:
Ah, I see. This would need an update on the bcc.Runtime interface. With typing, it throws an error.

Reply:
Was just a quick recommendation. We should not change bcc.Runtime interface for that. ProfileOptions are different from the runtime, but basically just a reduced runtime, including the `defaultCryptoAlgo` param. We can just force the runtime to be applied to the profile options.